components/script/dom/attr.rs

```diff
@@ -181,11 +36,13 @@ impl Attr {
                      prefix: Option<Atom>, owner: Option<&Element>) -> Attr {
         Attr {
             reflector_: Reflector::new(),
-            local_name: local_name,
+            name: AttrName {
+                local_name: local_name,
+                name: name,
+                namespace: namespace,
+                prefix: prefix,
+            },
```

Comment:
Could you name this field and structure differently? `foo.foo` also sounded silly to me. Maybe `AttrID`, or `AttrIdentity`, or `AttrInfo`, or something entirely different that I can't think of?